src/server/network/protocol/protocolgame.cpp

```diff
@@ -3923,7 +3923,7 @@ void ProtocolGame::sendBlessStatus() {
 	if (oldProtocol) {
 		msg.add<uint16_t>(blessCount >= 5 ? 0x01 : 0x00);
 	} else {
-		bool glow = g_configManager().getBoolean(INVENTORY_GLOW) || player->getLevel() < g_configManager().getNumber(ADVENTURERSBLESSING_LEVEL);
+		bool glow = (blessCount >= 5 || player->getLevel() < g_configManager().getNumber(ADVENTURERSBLESSING_LEVEL)) && g_configManager().getBoolean(INVENTORY_GLOW);
```

Comment:
We have 2 cases:

Like global, the glow is only before the player reaches `ADVENTURERSBLESSING_LEVEL`
```cpp
bool glow = player->getLevel() < g_configManager().getNumber(ADVENTURERSBLESSING_LEVEL) && g_configManager().getBoolean(INVENTORY_GLOW);
```
Or we can glow before the player reaches `ADVENTURERSBLESSING_LEVEL` OR we enable if player has 5+ blesses.
```cpp
bool glow = player->getLevel() < g_configManager().getNumber(ADVENTURERSBLESSING_LEVEL) || (g_configManager().getBoolean(INVENTORY_GLOW) && blessCount >= 5);
```

Reply:
I think we can use the second option, following the global behavior about the adventurer blessing and still keeping an "custom" way to enable it by config.lua, if you want

Reply:
but the way i wrote isnt right?

> We have 2 cases:
> 
> Like global, the glow is only before the player reaches `ADVENTURERSBLESSING_LEVEL`
> 
> ```c++
> bool glow = player->getLevel() < g_configManager().getNumber(ADVENTURERSBLESSING_LEVEL) && g_configManager().getBoolean(INVENTORY_GLOW);
> ```
> 
> Or we can glow before the player reaches `ADVENTURERSBLESSING_LEVEL` OR we enable if player has 5+ blesses.
> 
> ```c++
> bool glow = player->getLevel() < g_configManager().getNumber(ADVENTURERSBLESSING_LEVEL) || (g_configManager().getBoolean(INVENTORY_GLOW) && blessCount >= 5);
> ```

Reply:
I understand, both statements will give the same value to the variable glow based on the conditions, but they handle the evaluation of these conditions differently (in the order of evaluation only).